wifi: Add nRF7001DK support to lock sample (#14583)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

    You must use the alias to specify the QDEC instance used for scroll wheel, if your board supports multiple QDEC instances (for example ``nrf54l15pdk_nrf54l15_cpuapp``).
    You do not need to define the alias if your board supports only one QDEC instance, because in that case, the wheel module can rely on the ``qdec`` DT label provided by the board.

> * Updated the :kconfig:option:`CONFIG_BT_ADV_PROV_TX_POWER_CORRECTION_VAL` Kconfig option value in the nRF52840 Gaming Mouse configurations with the Fast Pair support.

Reviewer: Why is this line getting removed? Merge conflict?

Reviewer 2: Oh, that's weird... There was no merge conflict at all, so Git removed it after the rebase.

  * The :file:`prj_no_dfu.conf` file.
  * Support for ``no_dfu`` build type for nRF5350 DK, nRF52840 DK and nRF7002 DK.

* Added:

   * Test event triggers to all Matter samples.
     By utilizing the test event triggers, you can simulate various operational conditions and responses in your Matter device without the need for external setup.

     To get started with using test event triggers in your Matter samples and to understand the capabilities of this feature, refer to the :ref:`ug_matter_test_event_triggers` page.

>    * Support for emulation of nRF7001 Wi-Fi companion IC on nrf7002 DK to Matter Lock Sample.

Reviewer: ```suggestion

* :ref:`matter_lock_sample` sample:

   * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
```
So far, we've had changes specific to a single sample under an entry for that sample.
